cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -625,8 +624,15 @@ protected void startProgram(ProgramId program, int expectedStatusCode)
                                 program.getVersion(),
                                 program.getType().getCategoryName(),
                                 program.getProgram());
-    HttpResponse response = doPost(getVersionedAPIPath(path, program.getNamespace()),
-                                   GSON.toJson(ImmutableMap.<String, String>of()));
+    startProgram(path, program.getNamespace(), ImmutableMap.<String, String>of(), expectedStatusCode);
+  }
+
+  /**
+   * Tries to start the given program with the given runtime arguments and expect the call completed with the status.
+   */
+  protected void startProgram(String path, String namespaceId, Map<String, String> args, int expectedStatusCode)
```

Comment:
code formatting guideline: you have both the last parameter 'expectedStatusCode' and ``throws Exception`` on the second line, so that it is clear what exceptions that method is throwing.